skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -84,6 +89,114 @@ void MpActor::SendToUser(const void* data, size_t size, bool reliable)
     throw std::runtime_error("sendToUser is nullptr");
 }
 
+std::unordered_map<std::string, std::unordered_map<int, std::vector<uint32_t>>>
+  lootTable = {
+    { "WEAP",
```

Comment:
enum class LootboxItemType with Weapon, Armor, Clothing instead of record-style names WEAP, etc.